plugin/nginx_plus_collectd.py

```diff
@@ -678,16 +678,15 @@ def __init__(self, status_host=None, status_port=None, username=None, password=N
         self.status_port = status_port or 8080
         self.auth_tuple = (username, password) if username or password else None
 
-        self.base_status_url = 'http://{}:{}/status'.format(self.status_host, str(self.status_port))
-        self.nginx_version_url = '{}/nginx_version'.format(self.base_status_url)
-        self.address_url = '{}/address'.format(self.base_status_url)
-        self.caches_url = '{}/caches'.format(self.base_status_url)
-        self.server_zones_url = '{}/server_zones'.format(self.base_status_url)
-        self.upstreams_url = '{}/upstreams'.format(self.base_status_url)
+        self.base_status_url = 'http://{}:{}/api/7'.format(self.status_host, str(self.status_port))
```

Comment:
Makes sense. I'll make changes accordingly to support both Legacy and New API.

Reply:
Hello @rmfitzpatrick,

We have done the changes to support both Legacy and New APIs. Please let us know in case of any queries or changes if required.